.travis.yml

```diff
@@ -5,10 +5,16 @@ go:
 
 services:
   - docker
+  - postgresql
 
 sudo: false
 
+before_script:
+  - psql -c 'create database docker_test;' -U postgres
+
 install:
+  - go get -u github.com/jteeuwen/go-bindata/...
+
   # not using gb atm as it does not have support for static linking
   #- go get github.com/constabulary/gb/...
```

Comment:
Not a big expert on compiled languages. What does this mean? Is there some C code we need to compile into the binary?